edge-middleware/bot-protection-datadome/lib/datadome.ts

```diff
@@ -105,18 +121,16 @@ export default async function datadome(req: NextRequest) {
 
       if (dataDomeRes.status !== 200) {
```

Comment:
Why do we check the value as we are in the case `dataDomeRes.status == 400` ?

Reply:
split in github is not clear enough, but the part you're missing is:
```
case 200:
case 301:
case 302:
case 401:
case 403:
```